src/main.cpp

```diff
@@ -113,8 +114,8 @@ void usage()
   std::cerr << std::endl;
   std::cerr << "TROUBLESHOOTING OPTIONS:" << std::endl;
   std::cerr << "    -v                      verbose messages" << std::endl;
-  std::cerr << "    -d                      (dry run) debug info" << std::endl;
-  std::cerr << "    -dd                     (dry run) verbose debug info" << std::endl;
+  std::cerr << "    -d STAGE                (dry run) debug info for various stages of bpftrace execution" << std::endl;
```

Comment:
Could just `-d` default to all?

Reply:
My original idea was the same but there are problems with that. First of all, it goes against [POSIX Utility Syntax Guidelines](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html#tag_12_02) which says:

> Option-arguments should not be optional.

Also, if we made the argument to `-d` optional, I think that we'd have to always use the argument without a space, i.e. `-dast`, `-dcodegen`, etc. Otherwise, optarg wouldn't know if the following part of the CLI is an argument or a next option.